public/javascripts/Admin/src/Admin.js

```diff
@@ -2,7 +2,8 @@ function Admin (_, $, c3, turf) {
     var self = {};
     self.markerLayer = null;
     self.auditedStreetLayer = null;
-    self.visibleMarkers = ["CurbRamp", "NoCurbRamp", "Obstacle", "SurfaceProblem", "Occlusion", "NoSidewalk", "Other"];
+    self.visibleMarkers = {"CurbRamp" : [1,2,3,4,5], "NoCurbRamp" : [1,2,3,4,5], "Obstacle" : [1,2,3,4,5],
```

Comment:
You can put [1, 2, 3, 4, 5] into a variable, say, severityList and use that in the visibleMarkers list.